components/ProfilePage/ProfileHeader.tsx

```diff
@@ -94,20 +52,21 @@ export const ProfileHeader = ({
         className={`d-flex flex-row justify-content-start align-items-center gap-3 mx-5`}
       >
         <ProfileIcon role={role} large />
-        <div>
-          <ProfileDisplayName className={`col-3 col-md-auto`}>
-            {profile.fullName}
-          </ProfileDisplayName>
-          {user && !isUser ? (
-            <FollowUserButton profileId={profileId} />
-          ) : (
+        <div className={`d-grid col-6 col-md-10 gap-2`}>
+          <ProfileDisplayName>{profile.fullName}</ProfileDisplayName>
+          {user && isUser ? ( // Am I Logged In? and Is This My Profile?
```

Comment:
Love the extra clarity with these comments!

It'd be nice if we could clean up the pre-existing naming here in the future - `isCurrentUser` on the `ProfilePage` gets passed into this component as `isUser` and I feel like `isCurrentUser` is a lot more descriptive of what this variable actually means (as it stands, my first instinct on reading this file was to interpret `isUser` as "Is this a user or an organization?"). That might obviate the need for these extra comments by injecting the extra clarity into the variable name itself).